backend/bittan/bittan/tests/views/test_views.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase, Client
+from rest_framework import status
+from django.utils import timezone
+import datetime
+from bittan.models import ChapterEvent
+
+class GetChaptereventsTest(TestCase):
+
+	def setUp(self):
+		pass
+
+	def test_empty(self):
+		c = Client()
+		response = c.get("/get_chapterevents/")
+		self.assertEqual(response.status_code, status.HTTP_200_OK)
+		data = response.json()
+		self.assertEqual(data, [])
+	
+	def test_filled(self):
+		now = timezone.now()
+		ChapterEvent.objects.create(title="Title1", description="Description1", max_tickets=10, sales_stop_at=now+datetime.timedelta(days=365), event_at=now+datetime.timedelta(days=2))
+		ChapterEvent.objects.create(title="Title2", description="Description2", max_tickets=10, sales_stop_at=now+datetime.timedelta(days=365), event_at=now+datetime.timedelta(days=1))
+		c = Client()

```

Comment:
Get the real id:s from the created events and use them to check that the correct id:s in the test instead of only testing the type.

Reply:
Fixed as of e65977998dc926f1391a84e117a2d697b34f4d6c